insights/specs/default.py

```diff
@@ -380,13 +380,16 @@ def httpd_cmd(broker):
     ls_ocp_cni_openshift_sdn = simple_command("/bin/ls -l /var/lib/cni/networks/openshift-sdn")
     ls_origin_local_volumes_pods = simple_command("/bin/ls -l /var/lib/origin/openshift.local.volumes/pods")
     ls_osroot = simple_command("/bin/ls -lan /")
-    ls_run_systemd_generator = simple_command("/bin/ls -lan /run/systemd/generator")
+    ls_run_systemd = simple_command("/bin/ls -lanRL /run/systemd")
     ls_R_var_lib_nova_instances = simple_command("/bin/ls -laR /var/lib/nova/instances")
     ls_sys_firmware = simple_command("/bin/ls -lanR /sys/firmware")
     ls_tmp = simple_command("/bin/ls -la /tmp")
     ls_usr_bin = simple_command("/bin/ls -lan /usr/bin")
     ls_usr_lib64 = simple_command("/bin/ls -lan /usr/lib64")
     ls_usr_lib_systemd = simple_command("/bin/ls -lanRL /usr/lib/systemd")
+    ls_usr_local_lib_systemd = simple_command("/bin/ls -lanRL /usr/local/lib/systemd")
```

Comment:
@jobselko Could you please remove the specs that the parsers aren't be added yet. If a spec is added to core it needs a parser added with it, otherwise we're collecting something that can't be parsed.

Reply:
See the discussion on the spec MR.